cmd/rook/ceph/cleanup.go

```diff
@@ -117,3 +147,53 @@ func secretKeyMatch(monDir, monSecret string) (bool, error) {
 
 	return monSecret == extractedKey, nil
 }
+
+func wipeOSDDisk(context *clusterd.Context) {
+	// LVM based OSDs
+	osdLVMList, err := osd.GetCephVolumeLVMOSDs(context, clusterName, clusterFSID, "", false, false)
+	if err != nil {
+		logger.Errorf("failed to list lvm osd(s)")
+	}
+	for _, osd := range osdLVMList {
+		err := wipeLVM(context, osd.ID)
+		if err != nil {
+			logger.Errorf("failed to wipe osd %d disk %q", osd.ID, osd.BlockPath)
+		}
+	}
+
+	// Raw based OSDs
+	osdRawList, err := osd.GetCephVolumeRawOSDs(context, clusterName, clusterFSID, "", "", false)
+	if err != nil {
+		logger.Errorf("failed to list raw osd(s)")
+	}
+	for _, osd := range osdRawList {
+		logger.Infof("zero-ing osd %d disk %q", osd.ID, osd.BlockPath)
+		err := wipeDisk(context, osd.BlockPath)
+		if err != nil {
+			logger.Errorf("failed to wipe osd %d disk %q", osd.ID, osd.BlockPath)
+		}
+	}
+}
+
+func wipeDisk(context *clusterd.Context, blockPath string) error {
+	ddCommand := fmt.Sprintf("dd if=%s of=%s bs=%s count=%s oflag=%s", ddIf, blockPath, ddBS, ddCount, ddFlags)
```

Comment:
Don't the `dd` args need to be separate strings similar to the c-v args on line 191?